cityofaustin/atd-vz-data
# ETL: Download/Extract ZIP

Closes #460 

How to test this script:
1. Update your staging and production env files, there are two new variables needed there.
2. Run: `source runetl.sh`
3. Run: `runetl build`
4. Run: `runetl [path to staging env file] app/process_cris_request_download.py`

What to expect:
In your `tmp/` folder you should see some files like these:
![Screen Shot 2019-11-20 at 5 22 18 PM](https://user-images.githubusercontent.com/5282430/69287023-65534500-0bba-11ea-8d30-9968ae747060.png)

1. The `email_` are raw emails stored in s3 and are now downloaded to the folder for processing.


2. If you see zip files, that means the download process succeeded.

3. Check your `data/` folder, if you see csv files there, it means the extraction of those files succeeded.

** OPTIONAL
4. Check the finished folder in s3, [you should see three folders](https://s3.console.aws.amazon.com/s3/buckets/atd-vision-zero-editor/txdot-email-finished/?region=us-east-1&tab=overview).
